src/docs/markdown/caddyfile/directives/tls.md

```diff
@@ -359,23 +359,22 @@ Obtains certificates using the ACME protocol. Note that `acme` is a default issu
 
 	- **any_common_name** <span id="any_common_name"/> is a list of one or more common names; Caddy will choose the first chain that has an issuer that matches with at least one of the specified common names.
 
+##### ZeroSSL
```

Comment:
We can remove this section and just move the two paragraphs to under the `zerossl` header below.

Reply:
The rationale for the seemingly redundant heading was grouping under the ACME issuer module to which those notes pertain to. Aren't they technically separate issuer modules for the same issuer?

Reply:
Might make sense to mention ZeroSSL can be enabled by using the `acme` issuer (and putting the ZeroSSL URL in that section too) and leave the `zerossl` issuer as explaining it's for the API and not ACME, pointing back to `acme` for that. Maybe.

Reply:
I removed the subsection but retained the ACME-specific configuration tips under the `acme` issuer module.